extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/usecase/model/TransferprocessDto.java

```diff
@@ -30,4 +30,16 @@
 public class TransferprocessDto {
     @Schema(description = "ID of transferprocess")
     private String id;
+
+    @Schema(description = "transferprocess ceated at")
+    private long createdAt;
+
+    @Schema(description = "transferprocess updated at")
+    private long updatedAt;
+
+    @Schema(description = "transferprocess state")
+    private int state;
```

Comment:
needs to be a decent data type, e.g. an enum or an object "id, label, preferrably even description"